python/ccxt/huobipro.py

```diff
@@ -279,7 +279,89 @@ def fetch_balance(self, params={}):
             account['total'] = self.sum(account['free'], account['used'])
             result[currency] = account
         return self.parse_balance(result)
+    
+    def fetch_orders(self, symbol=None, since=None, limit=None, params={}):
+        if not symbol:
+            raise ExchangeError(self.id + 'fetchOrders requires a symbol parameter')
+        self.load_markets()
+        market = self.market(symbol)
+
+        if 'type' in params:
+            status = params['type']
+        elif 'status' in params:
+            status = params['status']
+        else:
+            raise ExchangeError(self.id + ' fetchOrders() requires type param or status param for spot market ' + symbol + '(0 or "open" for unfilled or partial filled orders, 1 or "closed" for filled orders)')
+        if status in [0,'open']:
+            status='submitted,partial-filled'
+        elif status in [1,'closed']:
+            status='filled,partial-canceled'
+        else:
+            raise ExchangeError(self.id + ' fetchOrders() wrong type param or status param for spot market ' + symbol + '(0 or "open" for unfilled or partial filled orders, 1 or "closed" for filled orders)')
+        response = self.privateGetOrderOrders(self.extend({
+            'symbol': market['id'],'states':status,
```

Comment:
I believe, `states` in this line is a typo, apart from that, everything seems ok, converting it to JS now.

Reply:
please see below huopri API reference, 'states' is the parameter for the orders status. 
![image](https://user-images.githubusercontent.com/10844648/33752408-94ba1a82-dc1c-11e7-9bc0-b759b3e6dc5b.png)

Reply:
This indeed looks like a typo in their docs, however, if you did test it and it worked ok, then this must be a typo in their engine API... Did it actually work with `statEs` for you?

Reply:
yes, I have tested it. using 'states' is okay to fetch orders

Reply:
Got it, thanks for clarification! Merging this shortly.